fetchai/ledger/api/contracts.py

```diff
@@ -14,30 +16,17 @@
 class ContractsApi(ApiEndpoint):
     API_PREFIX = 'fetch.contract'
 
-    def create(self, owner: Entity, contract: 'Contract', fee: int, shard_mask: BitVector = None):
+    def create(self, owner: Entity, contract: 'Contract', fee: int, signers: Optional[List[int]] = None,
+               shard_mask: BitVector = None):
         ENDPOINT = 'create'
 
         logging.debug('Deploying contract', contract.address)
 
-        # Default to wildcard shard mask if none supplied
-        if not shard_mask:
```

Comment:
We have also gotten rid of default fallback shardmask (lines 23 to 25). If desired close this comment.